oktapam/constants/descriptions/attributes.go

```diff
@@ -49,6 +49,7 @@ var (
 	CreatedByUser                         = "The ASA User that created the resource."
 	CSRDetails                            = "Certificate Signing Request (CSR) Details."
 	DefaultAddress                        = "Default address of the gateway."
+	DelegatedAdminGroups                  = "The ids of the groups for users who are designated as delegated admins for the resource group"
```

Comment:
Left comment in other file but repeating here; don't think we should call these delegated admin groups but owners instead to match UI. If must have this name, then add text to description calling out that it corresponds to the Owners field of a resource group.